src/utils/dates.js

```diff
@@ -10,3 +10,10 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(newerPurchase, olderPurchase) {
```

Comment:
For calculation functions like this, it's nice to specify what kind of units your arguments are supposed to be. Either in comments or in your variables names, let us know that these date arguments should be in milliseconds.

Reply:
Copy that! Updated to include the units in the variable names